wrappers/wrapper.rb

```diff
@@ -915,6 +922,200 @@ def simplify_vehicle_duration(vrp, _result = nil, options = { mode: :simplify })
       nil
     end
 
+    def simplify_vehicle_pause(vrp, result = nil, options = { mode: :simplify })
+      # Simplifies vehicle pauses if there is no reason to keep them -- i.e., no services with timewindows
+      case options[:mode]
+      when :simplify
+        return nil unless !vrp.scheduling? &&
+                          vrp.relations&.none?{ |r| r.type == :maximum_duration_lapse } &&
+                          vrp.services&.none?{ |service|
```

Comment:
I do the dev only on relation[:shipment] in mind since we are doing the shipment -> 2service+relation switch. There is already so much missing from vrp.shipment that I don't see the point on beating a dead horse. 

If I am not mistaken we should have a ticket in v1.8 for this, if not we should move it to v1.8 because neither side (relation[:shipment] and vrp.shipment) is completely working at the moment (but  at least thanks to mt-web usage relation[:shipment] is in pretty good shape).

Reply:
I moved this to v1.8 : https://gitlab.com/mapotempo/optimizer-api/-/issues/714